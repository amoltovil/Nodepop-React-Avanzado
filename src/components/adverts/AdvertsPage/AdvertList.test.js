import React from 'react';
import { shallow } from 'enzyme';

import AdvertList from './AdvertsList';

describe('AdvertList', () => {
    const props = {
        adverts: []
    };
    
    const render = () => shallow(<AdvertList {...props} />);

    test('should render', () => {
        const wrapper = render();
        //console.log(wrapper.debug());
        expect(wrapper.exists()).toBe(true);
        expect(wrapper.find('ul').props().className).toBe('advertsList');

    });

    test('snapshot testing', () => {
        const wrapper = render();
        expect(wrapper).toMatchSnapshot();
    });
});