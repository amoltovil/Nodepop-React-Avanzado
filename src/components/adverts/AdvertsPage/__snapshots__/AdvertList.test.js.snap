// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdvertList snapshot testing 1`] = `
<ul
  className="advertsList"
/>
`;
